lib/atomic/feed.ex

```diff
@@ -0,0 +1,117 @@
+defmodule Atomic.Feed do
+  @moduledoc """
+  The Feed context.
+  """
+  use Atomic.Context
+
+  alias Atomic.Feed.Post
+
+  @doc """
+  Returns a cursor-based pagination of posts.
+  Learn more about why using cursor-based pagination at: https://use-the-index-luke.com/no-offset
+
+  ## Examples
+
+      iex> list_posts()
+      %{entries: [%Post{}], metadata: %{...}}
+
+  """
+  def list_posts(opts \\ []) do
```

Comment:
```suggestion
  def list_paginated_posts(opts \\ []) do
```
And create a simpler function called `list_posts`.
(Maybe there's a way to fuse this with the "next" one, by using named parameters)